### ISSUE TITLE
Switch back to Select Mode after doing transform operation

### ISSUE THREAD

**Issue by sa-mendez** [BlenderDefender/Gizmodal-Ops#14]
This may or may not be a feature request as maybe I simply dont know how to do this.  Therefore Ill ask my question first.

Is there a way to have gizmodal switch back to select mode after I perform the transform operation? Is there some setting either in gizmodal or blender that I need to enable?

Gizmodal is great, but for me it is very annoying to have to go back to select mode (especially If I am in edit mode) after doing any kind of transformation operation).  I am aware that this is the case when using the transform operations without having Gizmodal installed, so this may be a blender limitation.

Thank you.

**Comment by BlenderDefender** [comment]
@sa-mendez I think what you are looking for is the shortcut `w`. This enables the select tool in the toolbar, completely independent from Gizmodal Ops.

**Comment by sa-mendez** [comment]
@BlenderDefender Yeah, I have been using that , but was just wondering if there was setting within Gizmodal ops to automatically set it back to the selection tool after the transformation.

The fact is that without Gizmodal enabled, I could do the modal ops (e.g.  press G, followed by Y) do my move translation and upon finishing, I am back to select mode (since I never left it).  With Gizmodal, doing the same thing keeps me in the move tool, and I have to press ` w ` to go back to selection mode tool.  

Was wondering if there is a way to mimic this behavior from base blender  with Gizmodal.

**Comment by BlenderDefender** [comment]
@sa-mendez If I understand you correctly, you'd like to go back to select mode/the previously enabled tool after using the modal operator, just like in vanilla Blender, but stay in the Gizmo, if you're not using the modal operator?

**Comment by sa-mendez** [comment]
@BlenderDefender  Yeah, that is correct.  I think that would make things more consistent with vanilla blender, while retaining all of the advantages of Gizmodal.  I would also think that you would want to consider this as well for a vertex slide (GG) in edit mode or trackball rotation (RR).

**Comment by BlenderDefender** [comment]
@sa-mendez Alright, I've edited your original message to fully describe your feature request. Feel free to correct anything that I didn't get right.

**Comment by sa-mendez** [comment]
@BlenderDefender Yes, that is a much better description than my original.  Thank you very much.

**Comment by BlenderDefender** [comment]
Alright, I'll consider your feature request, but regarding the effort it's likely going to cause, the chances of it getting implemented aren't really high. If your feature request gets support by quite a few people, the chances increase, but for the moment, I'll let it exist here as a possibility.

**Comment by sa-mendez** [comment]
Ok, understood.  I appreciate your consideration of this as a possibility.

**Comment by xan2622** [comment]
I second sa-mendez's suggestion. It would be convenient if Gizmodal Ops switched back to the previous tool (such as the selection tool).
